core/src/main/java/com/spotify/metrics/core/SemanticMetricBuilder.java

```diff
@@ -68,17 +59,8 @@ public boolean isInstance(final Metric metric) {
         }
     };
 
-    SemanticMetricBuilder<Timer> TIMERS = new SemanticMetricBuilder<Timer>() {
-        @Override
-        public Timer newMetric() {
-            return new Timer();
```

Comment:
This constructor uses `ExponentiallyDecayingReservoir` : https://github.com/dropwizard/metrics/blob/v4.0.2/metrics-core/src/main/java/com/codahale/metrics/Timer.java#L58